### ISSUE TITLE
zq: add flag to not stop all readers upon one reader erroring

### ISSUE THREAD

**Issue by henridf** [brimsec/zq#578]
For consistency with the ingest behavior covered in #528

**Comment by philrz** [comment]
@henridf: Is this one already done?

**Comment by henridf** [comment]
yes. (i had the magic "closes #578" invocation in the PR but this didn't auto close 🤔)

**Comment by philrz** [comment]
Verified in `zq` commit `adc4e53`.

As described in the linked PR, the default behavior has not changed. For instance, let's say I start from a directory full of logs generated by Zeek, which often contains files that are not actual Zeek TSV logs. If I attempt a super quick read on all the files in the directory, it still exits with an error as soon as it fails to read one.

```
/usr/local/zeek/logs/2020-02-12$ ls
Port.08:02:29-08:14:02.log.gz		packet_filter.08:02:29-08:14:02.log.gz
capture_loss.08:02:28-08:14:03.log.gz	reporter.08:14:02-08:14:02.log.gz
conn-summary.08:02:41-08:14:02.log.gz	smtp.08:04:54-08:14:02.log.gz
conn-summary.08:14:02-08:14:03.log.gz	smtp.08:14:02-08:14:03.log.gz
conn.08:02:41-08:14:02.log.gz		software.08:13:07-08:14:02.log.gz
conn.08:14:02-08:14:03.log.gz		ssl.08:03:21-08:14:02.log.gz
dns.08:02:54-08:14:02.log.gz		stats.08:02:29-08:14:02.log.gz
dns.08:14:02-08:14:03.log.gz		stderr.08:02:28-08:14:03.log.gz
files.08:03:21-08:14:02.log.gz		stdout.08:02:28-08:14:03.log.gz
http.08:13:07-08:14:02.log.gz		weird.08:09:11-08:14:02.log.gz
known_hosts.08:03:21-08:14:02.log.gz	x509.08:03:21-08:14:02.log.gz
loaded_scripts.08:02:29-08:14:02.log.gz

/usr/local/zeek/logs/2020-02-12$ zq "count() by _path" *
conn-summary.08:02:41-08:14:02.log.gz: malformed input
```

However, if I invoke the new `-e` flag correctly, I can get it to run the query on the readable ones and see warnings about the unreadable ones.

```
/usr/local/zeek/logs/2020-02-12$ zq -e=false "count() by _path" *
conn-summary.08:02:41-08:14:02.log.gz: malformed input
conn-summary.08:14:02-08:14:03.log.gz: malformed input
stderr.08:02:28-08:14:03.log.gz: malformed input
stdout.08:02:28-08:14:03.log.gz: malformed input
#0:record[_path:string,count:uint64]
0:[dns;484;]
0:[ssl;58;]
0:[Port;1;]
0:[conn;355;]
0:[http;3;]
0:[smtp;9;]
0:[x509;34;]
0:[files;86;]
0:[stats;3;]
0:[weird;1;]
0:[reporter;2;]
0:[software;1;]
0:[known_hosts;1;]
0:[capture_loss;1;]
0:[packet_filter;1;]
0:[loaded_scripts;412;]
```

Thanks @henridf!
